server/src/main/java/io/spine/server/delivery/memory/InMemoryShardedWorkRegistry.java

```diff
@@ -53,10 +53,17 @@ public final class InMemoryShardedWorkRegistry extends AbstractWorkRegistry {
     private final Map<ShardIndex, ShardSessionRecord> workByNode = newConcurrentMap();
 
     @Override
-    public synchronized Optional<ShardProcessingSession> pickUp(ShardIndex index, NodeId nodeId) {
+    public synchronized PickUpAck pickUp(ShardIndex index, NodeId nodeId) {
         return super.pickUp(index, nodeId);
     }
 
+    @Override
+    public void release(ShardSessionRecord session) {
+        ShardSessionRecord record = workByNode.get(session.getIndex());
+        // Clear the node ID value and release the session.
```

Comment:
Why do we need this comment inlined? Can we use `@implNote` on a method level instead?